Formula/libfabric.rb

```diff
@@ -26,6 +26,6 @@ def install
   end
 
   test do
-    system "#(bin}/fi_info"
+    system "#{bin}/fi_info"
```

Comment:
Can we also check some output while we're at it?

Reply:
I know very little about the software and don't know what most of the output is. What does it output on your system? I could do something like searching for `provider: sockets` as I imagine every system has a socket interface.

Reply:
Seems good to me, I have plenty of mentions of those too.